### PR TITLE
release 1.1.2 - canguro

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

The only change of this version: We revert module `wfmash` to v0.10.4, because the current releases are unstable, not documented, and I don't understand how to set the parameters properly.

It is currently unclear, when a new *stable* release will become available. People like @baozg or @AndreaGuarracino are  evaluating.

### CODE REVIEW

**Comment by subwaystation** [comment]
I am praying this is the last release for my evaluations. Else I will go **PUFF**.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.1.2 - canguro
+
+This release reverts the `wfmash` tool version to 0.10.4, because the current releases are unstable, not documented, and I don't understand how to set the parameters properly.
+It is currently unclear, when a new _stable_ release will become available. People like @baozg or @AndreaGuarracino are evaluating.
```
Comment (mashehu) [inline CHANGELOG.md:8]:
```suggestion
It is currently unclear, when a new _stable_ release will become available. @baozg or @AndreaGuarracino are evaluating.
```

**Review by heuermh — APPROVED** [review]

File: conf/modules.config [hunk 2]
```diff
@@ -139,8 +139,8 @@ process {
                 "${wfmash_sparse_map_cmd}",
                 params.wfmash_temp_dir         ? "-B ${wfmash_temp_dir}"             : "",
                 "-m",
-                "-2 ${params.wfmash_hg_filter_ani_diff}",
-                "--lower-triangular",
+//                "-2 ${params.wfmash_hg_filter_ani_diff}",
+//                "--lower-triangular",
```
Comment (mashehu) [inline conf/modules.config:143]:
```suggestion
```

File: modules/nf-core/wfmash/environment.yml
Comment (mashehu) [inline modules/nf-core/wfmash/environment.yml:1]: are you sure you want to delete this?

File: nextflow_schema.json [hunk 3]
```diff
@@ -122,8 +122,7 @@
                 },
                 "wfmash_n_mappings": {
                     "type": "integer",
-                    "default": 1,
-                    "description": "Number of mappings for each segment."
+                    "description": "Number of mappings for each segment. [default: n_haplotypes - 1]."
```
Comment (mashehu) [inline nextflow_schema.json:125]:
```suggestion
                    "description": "Number of mappings for each segment. [default: `n_haplotypes - 1`]."
```

**Review by mashehu — CHANGES_REQUESTED** [review]

File: conf/modules.config (hunk 2 above)
Comment (subwaystation) [reply conf/modules.config:143]: I want to keep this for future releases of `wfmash`. Would that be fine?

File: modules/nf-core/wfmash/environment.yml
Comment (subwaystation) [reply modules/nf-core/wfmash/environment.yml:1]: so for now: yes I would say

**Review by mashehu — APPROVED** [review]

**Pull request merged**